site2/docs/reference-cli-tools.md

```diff
@@ -14,9 +14,9 @@ Pulsar offers several command-line tools that you can use for managing Pulsar in
 * `pulsar-shell`
 * `bookkeeper`
 
-::: tip
+:::tip
 
-For the latest and complete information about command-line tools, including commands, flags, descriptions, and more information, see [Pulsar Reference](https://pulsar.apache.org/reference).
+For the latest and complete information about command-line tools, including commands, flags, descriptions, and more information, see [Pulsar Reference](/reference).
```

Comment:
IIRC, `/` should be enforced at the end, like `/reference/`.
BTW, there is another occurrence of `https://pulsar.apache.org/reference/` in the `reference-configuration.md` file.

Reply:
Let me revert this change here. We can do it in a separate PR.

Reply:
Reverted.